packages/bolt-connection/test/bolt/response-handler.test.js

```diff
@@ -142,7 +144,7 @@ describe('response-handler', () => {
   })
 
   function testDiagnosticRecord (diagnostic_record, expected_diagnostic_record) {
-    expect(diagnostic_record.OPERATION).toBe(expected_diagnostic_record.OPERATION)
+    expect(diagnostic_record.OPERATION).toBe(expected_diagnostic_record.OPERATION ?? '')
```

Comment:
The expected diagnostic record should be informed with the actual value, we should not to this defaulting here because it can be confusing.